pyproject.toml

```diff
@@ -21,6 +21,21 @@ disable = [
   "W1113",  # Keyword argument before variable positional arguments  (PEP-570 is Python 3.8)
 ]
 
+[tool.ruff]
+exclude = [
+    ".git/",
+    "modules/",
+    "node_modules/",
+    "pkg/",
+    "test/common/",
+    "test/verify/",
+    "tools/",
+]
+ignore = [
+    "E731",  # Do not assign a `lambda` expression, use a `def`
+]
+line-length = 155
```

Comment:
This happens in a later commit.

The idea was to add ruff, and then firm things up as we fix them.  See the commit message:

>  The initial configuration is chosen to avoid having errors straight away.  Fixes will follow.